bounties/python/fsociety/1/vulnerability.json

```diff
@@ -0,0 +1,52 @@
+{
+  "PackageVulnerabilityID": "1",
+  "DisclosureDate": "2020-06-30",
+  "AffectedVersionRange": "*",
+  "Summary": "RCE on \"fsociety\"",
+  "Author": {
+    "Username": "Mik317",
+    "Name": "Michele Romano"
+  },
+  "Package": {
+    "Registry": "python",
+    "Name": "fsociety",
+    "URL": "https://github.com/Manisso/fsociety/"
+  },
+  "CWEs": [
+    {
+      "ID": "78",
+      "Description": "OS command injection"
+    }
+  ],
+  "CVSS": {
+    "Version": "3.1",
+    "AV": "",
+    "AC": "",
+    "PR": "",
+    "UI": "",
+    "S": "",
+    "C": "",
+    "I": "",
+    "A": "",
+    "E": "",
+    "RL": "",
+    "RC": "",
+    "Score": "6.4"
+  },
+  "CVEs": [],
+  "Repository": {
+    "URL": "https://github.com/Manisso/fsociety/",
+    "Codebase": [
+      "Python"
+    ]
+  },
+  "Permalinks": [
+    "https://github.com/Manisso/fsociety/blob/master/fsociety.py#L485",
+    " https://github.com/Manisso/fsociety/blob/master/fsociety.py#L488",
+    " https://github.com/Manisso/fsociety/blob/master/fsociety.py#L491",
+    " https://github.com/Manisso/fsociety/blob/master/fsociety.py#L583",
+    " https://github.com/Manisso/fsociety/blob/master/fsociety.py#L587",
+    " https://github.com/Manisso/fsociety/blob/master/fsociety.py#L636"
```

Comment:
Remove spaces at the beginning of the strings

Reply:
Done